src/components/sidebar.js

```diff
@@ -47,50 +49,87 @@ const NavItem = ({ name, path }) => {
 
 const SocialLink = ({ icon }) => <li className="mx-1">{icon}</li>;
 
-const SideBar = ({ renderAside, setRenderAside }) => (
-  <aside
-    className={
-      renderAside
-        ? 'top-0 absolute bg-slate-100 lg:relative flex flex-col justify-between h-screen w-full lg:w-1/6 border-r-2 py-6'
-        : 'hidden'
-    }
-  >
-    <div className="lg:hidden absolute right-2 top-2 border">
-      <button className="p-1" type="button" onClick={() => setRenderAside()}>
-        <MdOutlineClose className="text-xl text-gray-700" />
-      </button>
-    </div>
-    <div>
-      {' '}
-      <div className=" hidden lg:flex justify-center items-center">
-        <span className="h-28 block">Logo</span>
+const SideBar = ({ renderAside, setRenderAside }) => {
+  const userState = useSelector((state) => state.user);
+  const dispatch = useDispatch();
+
+  const deleteToken = () => {
+    localStorage.removeItem('token');
+  };
+
+  const removeCurrentUser = () => {
+    localStorage.removeItem('user');
+  };
+
+  const logout = () => {
+    deleteToken();
+    removeCurrentUser();
+    dispatch(logUserOut());
+  };
+
+  return (
+    <aside
+      className={
+        renderAside
+          ? 'top-0 absolute bg-slate-100 lg:relative flex flex-col justify-between h-screen w-full lg:w-1/6 border-r-2 py-6'
+          : 'hidden'
+      }
+    >
+      <div className="lg:hidden absolute right-2 top-2 border">
+        <button className="p-1" type="button" onClick={() => setRenderAside()}>
+          <MdOutlineClose className="text-xl text-gray-700" />
+        </button>
       </div>
-      <ul className="pl-5 pt-9 lg:pt-0">
-        {links.map((link) => (
-          <NavItem
-            key={links.indexOf(link)}
-            name={link.name}
-            path={link.path}
-          />
-        ))}
-      </ul>
-    </div>
-    <div>
-      <ul className="flex justify-center mb-3">
-        {socialIcons.map((icon) => (
-          <SocialLink icon={icon} key={icon.key} />
-        ))}
-      </ul>
-      <span className="text-xs font-medium w-full text-center block">
-        ©
+      <div>
         {' '}
-        {new Date().getFullYear()}
-        {' '}
-        Booking App
-      </span>
-    </div>
-  </aside>
-);
+        <div className=" hidden lg:flex justify-center items-center">
+          <span className="h-28 block">Logo</span>
+        </div>
+        <ul className="pl-5 pt-9 lg:pt-0">
+          {links.map((link) => (
+            <NavItem
+              key={links.indexOf(link)}
+              name={link.name}
+              path={link.path}
```

Comment:
I think it'd be best if you hide `My Reservations` `Add Motorcycle` `Delete Motorcycle` and only allow logged in users to have access. This is because these operations need an authorized user in order to proceed.

![image](https://user-images.githubusercontent.com/29071984/169808399-21364bd1-a1cf-4b9d-ba5f-59ad978927ff.png)